src/components/inversionSolution/InversionSolutionHazardTab.tsx

```diff
@@ -20,19 +25,49 @@ const InversionSolutionHazardTab: React.FC<InversionSolutionHazardTabProps> = ({
   const options = getHazardTableOptions(data);
 
   const [location, setLocation] = useState<string>(options.location[0]);
-  const [PGA, setPGA] = useState<string>(options.PGA[0]);
+  const [PGA, setPGA] = useState<string[]>([options.PGA[0]]);
   const [forecastTime, setForecastTime] = useState<string>(options.forecastTime[0]);
   const [gmpe, setGmpe] = useState<string>(options.gmpe[0]);
   const [backgroundSeismicity, setBackgroundSeismicity] = useState<string>(options.backgroundSeismicity[0]);
 
-  const [filteredData, setFilteredData] = useState<XY[]>([]);
+  const [filteredData, setFilteredData] = useState<HazardTableFilteredData>({});
 
   useEffect(() => {
-    const pgaValue = PGA === 'PGA' ? '0.0' : PGA;
-    const xy = filterData(data, location, pgaValue, forecastTime, gmpe, backgroundSeismicity);
-    setFilteredData(xy);
+    const filtered: HazardTableFilteredData = {};
+    const pgaValues: string[] = [...PGA];
+    if (pgaValues.includes('PGA')) {
+      const index = pgaValues.indexOf('PGA');
+      pgaValues[index] = '0.0';
+    }
+
+    pgaValues.map((pgaValue) => {
+      const xy = filterData(data, location, pgaValue, forecastTime, gmpe, backgroundSeismicity);
+      pgaValue === '0.0' ? (filtered['PGA'] = xy) : (filtered[pgaValue] = xy);
+    });
+    setFilteredData(filtered);
   }, [location, PGA, forecastTime, gmpe, backgroundSeismicity]);
 
+  const width = 1400;
+  const height = 1000;
+  const marginLeft = 100;
+  const marginRight = 100;
+  const marginTop = 100;
+  const marginBottom = 100;
+
+  const xMax = width - marginLeft - marginRight;
+  const yMax = height - marginTop - marginBottom;
+
+  const handleSetPGA = (selections: string[]) => {
+    setPGA(selections);
+  };
+
+  const colors = ['#FE1100', '#73d629', '#ffd700', '#7fe5f0', '#003366', '#ff7f50', '#047806', '#4ca3dd'];
```

Comment:
It seems like the colours are limiting the number of series that can be displayed on the map at once, e.g. if you select more than 8, they don't show up. Maybe limit the number that can be selected in that case.

Reply:
Also consider whether none can be selected, or if we want to ensure at least one is always selected. Otherwise the entire chart disappears.

Reply:
Added alert and limiting number of curves able to be displayed